packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -110,7 +110,8 @@ export const Checkbox: React.FC<CheckboxProps> = ({
         <Label
           htmlFor={props.id}
           required={required}
-          variant={props.disabled ? 'disabled' : 'inline'}
+          variant="inline"
```

Comment:
Removing hard coded variant from checkbox is in one of the next PRs. 
This is just  to make the ckeckbox work for the moment.